Installations-HowTos/How-To-Install-Spark-On-MACOS.md

```diff
@@ -31,42 +30,42 @@ http://spark.apache.org/downloads.html
 
 Direct link is for example following 
 
-http://mirror.metrocast.net/apache/spark/spark-3.0.0/spark-3.0.0-bin-hadoop2.7.tgz
+https://dlcdn.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz
 
 You can run the following command to download: 
 
-```wget http://mirror.metrocast.net/apache/spark/spark-3.0.0/spark-3.0.0-bin-hadoop2.7.tgz```
+```wget https://dlcdn.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz```
 
 
 Uncompress it 
 
-```tar xvfz spark-3.0.0-bin-hadoop2.7.tgz```
+```tar xvfz spark-3.5.0-bin-hadoop3.tgz```
 
 I have my spark on the following folder 
 
-```/Users/kiat/spark-3.0.0-bin-hadoop2.7```
+```/Users/sandeep/spark-3.5.0-bin-hadoop3```
 
 # Step-3 -  Setup Enviroment Variables. 
 
 Edit the bash_profile file and set some Environment variables 
 
 ```nano .bash_profile```
 
-**Note:** For maxOS Catalina user, you need to edit the zshrc file and set the below enviroment variables to the end of your file 
+**Note:** For maxOS Catalina and above users, you need to edit the zshrc file and set the below enviroment variables to the end of your file 
 
 Also, if your zsh shell could not find the jupyter notebook command, here is the link to solve this problem
 
 https://medium.com/@sumitmenon/how-to-get-anaconda-to-work-with-oh-my-zsh-on-mac-os-x-7c1c7247d896
 
 ```nano .zshrc```
 
-```export SPARK_HOME=/Users/kiat/spark-3.0.0-bin-hadoop2.7```
+```export SPARK_HOME=/Users/sandeep/spark-3.5.0-bin-hadoop3```

```

Comment:
let make this more generic using place holder, such as /Users/<your-username>/

Reply:
added generic place holder